helpers/bot_client.py

```diff
@@ -290,6 +290,22 @@ async def flip(self, ctx, number=1):
         except Exception as e:
             await ctx.send(str(e))
 
+    @commands.command(name='roll', help='Rolls a dice of your choosing')
+    async def roll(self, ctx, number=1, dice_size="D6"):
+        total=0
+        try:
+            await ctx.send(f'Rolling a {dice_size} {number}time(s)...')
+            for i in range(number):
+                result = random.randint(1, int(dice_size.lower().replace("d","")))
+                total+=result
+                await ctx.send("You rolled a "+ str(result))
+            await asyncio.sleep(1)
+            await ctx.send(f'Your total rolls equal {total}')
+        except Exception as e:
+            await ctx.send(str(e))
+
+
+
```

Comment:
nit - remove 1 breakline here, 2 breaklines is the recommended amount for Python between methods outside of classes